frontend/projects/upgrade/src/app/shared-standalone-component-lib/components/common-section-card-action-buttons/common-section-card-action-buttons.component.html

```diff
@@ -38,10 +35,7 @@
       </button>
     </mat-menu>
 
-    <button
-      mat-icon-button
-      (click)="onSectionCardExpandChange()"
-    >
+    <button mat-icon-button [disabled]="primaryActionBtnDisabled" (click)="onSectionCardExpandChange()">
```

Comment:
I think it would be better not to have the expand button rely on the primary button's disabled state, in case we want to disable only the primary button later. We can add another input variable, such as `sectionCardExpandBtnDisabled`.